components/fieldgroups/profile/profile-healthcare-member.schema.json

```diff
@@ -87,10 +79,20 @@
               "description": "Emergency contact details.",
```

Comment:
Let's remove Emergency Contact since no one could think of a use case to use it.